src/Components/DataDisplay/ChipGroup.vue

```diff
@@ -33,16 +33,58 @@ export default {
 	&.small {
 		&::v-deep .c-chip {
 			margin-right: 4px;
+			&:last-child {
+				margin-right: 0;
+			}
+		}
+		&::v-deep .c-row {
+			.c-style-col {
+				padding-right: 4px;
+				&:last-child {
+					padding-right: 0;
+				}
+			}
 		}
 	}
 	&.medium {
 		&::v-deep .c-chip {
 			margin-right: 6px;
+			&:last-child {
+				margin-right: 0;
+			}
+		}
+		&::v-deep .c-row {
+			.c-style-col {
+				padding-right: 6px;
+				&:last-child {
+					padding-right: 0;
+				}
+			}
 		}
 	}
 	&.large {
 		&::v-deep .c-chip {
 			margin-right: 8px;
+			&:last-child {
+				margin-right: 0;
+			}
+		}
+		&::v-deep .c-row {
+			.c-style-col {
+				padding-right: 8px;
+				&:last-child {
+					padding-right: 0;
+				}
+			}
+		}
+	}
+	&::v-deep .c-row {
+		margin: 0;
+		.c-style-col {
+			padding: 0;
```

Comment:
```suggestion
			padding: 0;
			&:last-child {
				padding-right: 0;
			}
```
여기에 넣으면 `.small`, `.medium`, `.large`에 있는 코드 반복을 줄일 수 있겠네요